datasource.go

```diff
@@ -119,6 +120,28 @@ func (ds *SQLDatasource) QueryData(ctx context.Context, req *backend.QueryDataRe
 	// Execute each query and store the results by query RefID
 	for _, q := range req.Queries {
 		go func(query backend.DataQuery) {
+			defer wg.Done()
+
+			// Panic recovery
+			defer func() {
+				if r := recover(); r != nil {
+					stack := string(debug.Stack())
+					errorMsg := fmt.Sprintf("SQL datasource query execution panic: %v", r)
+
+					backend.Logger.Error(errorMsg,
+						"panic", r,
+						"query", query,
```

Comment:
Since we're planning to audit our code to remove any logging of potentially sensitive data, including queries, I think we should remove this.